src/TestSuite/TestCookieEncrypter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @since         3.0.0
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\TestSuite;
+
+use Cake\Controller\ComponentRegistry;
+use Cake\Controller\Component\CookieComponent;
+
+/**
+ * Class TestCookieEncrypter
+ *
+ * This class is to encrypt a cookie value in the IntegrationTestCase as if CookieComponent would do.
+ *
+ * @package Cake\TestSuite
+ */
+class TestCookieEncrypter extends CookieComponent
```

Comment:
Instead of subclassing perhaps we should extract the encryption/decryption logic out of the Component into a more re-usable piece of code?